### PR TITLE
upgrade debian virtualbox image

### PR DESCRIPTION
**Pull request by xmunoz**
Upgrade debian base image, and expand/upgrade documentation.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -68,6 +68,7 @@ Vagrant.configure(2) do |config|
           "AWS_ACCESS_SECRET": ENV["AWS_ACCESS_SECRET"],
           "AWS_REGION": ENV["AWS_REGION"],
           "PERM_ENV": "local",
+          "PERM_HOSTNAME": "localdev",
```
Comment (jasonaowen) [inline Vagrantfile:71]: Hm, I don't follow how this works; where is `PERM_HOSTNAME` used? I don't see it in `infrastructure` or in this repo.

Comment (xmunoz) [reply Vagrantfile:71]: This was from an older iteration of a change that I made to infrastructure, and is, in fact, no longer needed. Thanks for the keen eye! Will revert this and the change below, and change the PR title.

File: Vagrantfile [hunk 2]
```diff
@@ -15,7 +15,7 @@ Vagrant.configure(2) do |config|
   # Every Vagrant development environment requires a box. You can search for
   # boxes at https://atlas.hashicorp.com/search.
   config.vm.box = "generic/debian10"
-  config.vm.box_version = "3.0.22"
+  config.vm.box_version = "3.1.8"
```
Comment (jasonaowen) [inline Vagrantfile:18]: Fun times: [3.1.10](https://app.vagrantup.com/generic/boxes/debian10/versions/3.1.10)

> This version was created about 6 hours ago.

**Review by jasonaowen — COMMENTED** [review]
I'm testing this now!

Some helpful commands I had to learn when I ran out of disk space in my homedir:

```
vagrant box list
vagrant box remove {box_name}
```

**Review by jasonaowen — APPROVED** [review]
I successfully deployed with box version 3.1.8! I trust the point release contains no major changes, and don't really want to wait another half an hour to rebuild.

Please squash the "Update Vagrantfile" commit, either via the squash+merge button in the web UI or via the CLI + force push; it's not a useful commit to have in our history.

**Pull request merged**